implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -255,11 +249,16 @@ private DimensionList fromTags(List<Tag> tags) {
 
     private void send(List<String> metricLines) {
         try {
+            String endpoint = config.uri();
```

Comment:
Maybe we should consider adding a test, just to make sure it failed before the fix and now it does the correct thing.

Reply:
I have added a test in 78c0ed0